salt/modules/libcloud_dns.py

```diff
@@ -42,9 +42,9 @@
     from libcloud.dns.types import RecordType
 
     # pylint: enable=unused-import
-    if hasattr(libcloud, "__version__") and _LooseVersion(
-        libcloud.__version__
-    ) < _LooseVersion(REQUIRED_LIBCLOUD_VERSION):
+    if hasattr(libcloud, "__version__") and Version(libcloud.__version__) < Version(
```

Comment:
Can you help my understanding here. Why are we updating to `Version` in some modules but keeping Looseversion in others?

Reply:
Version is from packaging, which has nothing to do with StrictVersion.

Version from packaging expects properly formatted versions, while Looseversion is less strict and is actually required to parse some of the weird string versions.
I tried to limit the use of Looseversion to where it is only needed.